pathspider/plugins/evilbit.py

```diff
@@ -0,0 +1,111 @@
+from scapy.all import IP  # pylint: disable=E061
+from scapy.all import IPv6  # pylint: disable=E061
+from scapy.all import UDP  # pylint: disable=E061
+from scapy.all import TCP  # pylint: disable=E061
+from scapy.all import DNS  # pylint: disable=E061
+from scapy.all import DNSQR  # pylint: disable=E061
+from scapy.all import RandShort  # pylint: disable=E061
```

Comment:
All the disable=E601 need to be changed to "disable=no-name-in-module" as the pylint on Travis doesn't like the old-style disables.